src/main/java/com/openshift/internal/client/httpclient/UrlConnectionHttpClient.java

```diff
@@ -304,8 +313,12 @@ private void setAcceptHeader(String acceptedVersion, String acceptedMediaType, H
 		connection.setRequestProperty(PROPERTY_ACCEPT, builder.toString());
 	}
 
-	private void setAuthorization(String username, String password, String authKey, String authIV, String token,
+	protected final void setAuthorization(String username, String password, String authKey, String authIV, String token,
```

Comment:
here, too: I would think that there should be no alternative to the authorization strategy: if we want to use username/password, we'd have to provide an appropriate authorization strategy